Meta/build-root-filesystem.sh

```diff
@@ -81,6 +85,11 @@ chown 0:$utmp_gid mnt/var/run/utmp
 chmod 664 mnt/var/run/utmp
 echo "done"
 
+printf "create locate folder..."
```

Comment:
```suggestion
printf "createing locate folder... "
```

trailing space is on purpose as this doesn't print a newline, so the "done" below will go after the "...".

Reply:
small typo: "createing" -> "creating"

Reply:
whoops, thanks!